custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -180,6 +179,7 @@ def _get_episode_case_properties(episode_case_properties):
         ),
         "dateofInitiation": episode_case_properties.get('treatment_initiation_date', str(datetime.date.today())),
         "Ptype": patient_type_choice.get(episode_case_properties.get('patient_type_choice', ''), ''),
+        "sitedetail": site_detail
```

Comment:
will this fail if `episode_site_choice` is `None`? I don't think this variable gets set otherwise.